samples/javascript_nodejs/11.qnamaker/bots/QnABot.js

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+const { ActivityHandler } = require('botbuilder');
+const { QnAMaker } = require('botbuilder-ai');
+
+class QnABot extends ActivityHandler {
+    /**
+     * The QnAMakerBot constructor requires one argument (`endpoint`) which is used to create an instance of `QnAMaker`.
+     * @param {configuration} configuration The basic configuration needed to call QnA Maker. In this sample the configuration is retrieved from the .env file.
+     * @param {QnAMakerOptions} config An optional parameter that contains additional settings for configuring a `QnAMaker` when calling the service.
+     * @param {any} logger object for logging events, defaults to console if none is provided
+     */
+    constructor(configuration, qnaOptions, logger) {
```

Comment:
Let's get rid of `configuration` and `qnaOptions`, the code below will create the QnAMaker instance by pulling in the values from the environment variables.  Similar to how the LuisRecognizer is constructed in the [core-bot sample](https://github.com/Microsoft/BotBuilder-Samples/blob/samples-work-in-progress/samples/javascript_nodejs/13.core-bot/dialogs/luisHelper.js#L16-L20).

Can you drop `qnaOptions` for now since we're not using it in any meaningful manner? Its original value pre-feedback as an empty object with no explanation doesn't add anything.

Reply:
easy peasy